cmd/helm/installer/install.go

```diff
@@ -71,6 +72,11 @@ func Upgrade(client kubernetes.Interface, opts *Options) error {
 	}
 	obj.Spec.Template.Spec.Containers[0].Image = opts.SelectImage()
 	obj.Spec.Template.Spec.Containers[0].ImagePullPolicy = opts.pullPolicy()
+	if opts.MaxHistory > 0 {
```

Comment:
how would a user unset this value?

Reply:
Good point. If `opts.MaxHistory` is not set should we revert to the default of `0`?

@bacongobbler I'd appreciate hearing what you think is the best approach? Thanks!